### PR TITLE
Do not generate dark version of the view-icons w/ shadows

### PR DESCRIPTION
**Pull request by skjnldsv**
@nextcloud/designers 
Fix https://github.com/nextcloud/documentation/issues/1187

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]

**Comment by skjnldsv** [comment]
Bump!

File: core/css/icons.scss [hunk 1]
```diff
@@ -314,10 +314,25 @@ img, object, video, button, textarea, input, select, div[contenteditable='true']
...
 	filter: drop-shadow(1px 1px 4px var(--color-box-shadow));
 }
-
 @include icon-black-white('video-switch', 'actions', 1, true);
-@include icon-black-white('view-close', 'actions', 1, true);
-@include icon-black-white('view-download', 'actions', 1, true);
+
+/* SHADOW WHITE ICONS: white version only ----------------------------------- */
+.icon-view-close,
+.icon-view-close-white {
```
Comment (juliushaertl) [inline core/css/icons.scss]: I could not find any use of the -white classes, so I would just get rid of them and have only the icon-view-close,icon-view-download...

Comment (skjnldsv) [reply core/css/icons.scss]: Me neither, but they were here, so I did not wanted to break something :man_shrugging: :thinking:

Comment (juliushaertl) [reply core/css/icons.scss]: Ok, I'm also ok with keeping them. :woman_shrugging:

**Review by juliusknorr — APPROVED** [review]

**Comment by skjnldsv** [comment]
Failure unrelated:
```
    And I share "shared.txt" with "user0"                        # FilesAppSharingContext::iShareWith()
      │ Share with input in the details view in Files app value could not be set
      │ Exception message: Element is not currently visible and so may not be interacted with
      │ Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
      │ System info: host: 'e9db1e0d47ee', ip: '172.17.0.4', os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.0-43-generic', java.version: '1.8.0_91'
      │ Driver info: driver.version: unknown
      │ Trying again
      │ 

```

**Pull request merged**